app/controllers/steps/base_controller.rb

```diff
@@ -65,5 +66,9 @@ def ensure_session_integrity
     def initialize_commodity_context_service
       Thread.current[:commodity_context_service] = CommodityContextService.new
     end
+
+    def title
+      t("page_titles.#{@step.class.try(:id)}", default: super)
```

Comment:
I am wondering if we could create a method where @step is defined instead access it and call `step.class.try(:id)` here.

I just wanted to try a cleaner way, but not sure how at the moment.

Reply:
As far as I can make out the only place we need to do this is here so I don't mind too much asking the step for it's class just once. I reckon if this pattern repeats a bunch we can add an instance method on the step to pull out the id of the singleton, though